### PR TITLE
T16: Added create a comment scenario

### PR DESCRIPTION
**Pull request by adrianovi**

### CODE REVIEW

File: src/main/java/api/methods/APICategoriesMethods.java [hunk 1]
```diff
@@ -6,12 +6,14 @@
...
 import io.restassured.http.Headers;
 import io.restassured.response.Response;
+import utils.LoggerManager;
 import utils.StringManager;
 
 import java.util.HashMap;
 import java.util.Map;
 
 public class APICategoriesMethods {
+    public static final LoggerManager log = LoggerManager.getInstance();
```
Comment (Agustin-Mediotti) [inline src/main/java/api/methods/APICategoriesMethods.java:16]: Those instances are public

File: src/main/java/api/methods/APICommentsMethods.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import io.restassured.http.Header;
+import io.restassured.http.Headers;
+import io.restassured.response.Response;
+import utils.LoggerManager;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class APICommentsMethods {
+    public static final LoggerManager log = LoggerManager.getInstance();
```
Comment (Agustin-Mediotti) [inline src/main/java/api/methods/APICommentsMethods.java:15]: This ones too, if are the same fixed in the other PR then are going to have conflicts

**Review by Agustin-Mediotti — COMMENTED** [review]
There are some public instances

File: src/main/java/api/methods/APICommentsMethods.java (hunk 2 above)
Comment (adrianovi) [reply src/main/java/api/methods/APICommentsMethods.java]: Fixed

**Pull request merged**